ccd-definition/ComplexTypes.json

```diff
@@ -1398,5 +1398,37 @@
     "FieldType": "Document",
     "ElementLabel": "Upload a file",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "StandardDirections",
+    "ListElementCode": "standardDirectionsLabel",
+    "FieldType": "Label",
+    "ElementLabel": "Upload standard directions and other relevant documents, for example the C6 Notice of Proceedings or C9 statement of service.",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "StandardDirections",
+    "ListElementCode": "standardDirections",
+    "FieldType": "Document",
+    "ElementLabel": "Upload a file",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "StandardDirectionsOther",
```

Comment:
Please also rename to `CourtAdminDocument`. We would normally reuse `UploadDocument` but that has hint text on one of the properties that does not make sense in context of Court Admin documents :(